src/applications/travel-pay/redux/reducer.js

```diff
@@ -709,6 +620,12 @@ function travelPayReducer(state = initialState, action) {
         reviewPageAlert: null,
       };
 
+    case SET_EXPENSE_BACK_DESTINATION:
+      return {
+        ...state,
+        expenseBackDestination: action.payload,
+      };
```

Comment:
Can we add this within the `complexClaim` namespace in the store since it's specific to the complex claims flow?